pkg/chaintracker/chainfollower.go

```diff
@@ -9,7 +9,6 @@ import (
 )
 
 const (
-	DOGECOIN_GENESIS_BLOCK_HASH = "1a91e3dace36e2be3bf030a65679fe821aa1d6ef92e7c9902eb318182c355691"
```

Comment:
I would like to keep the mainnet and testnet genesis hashes for security, but also offer the option to auto-detect

Reply:
We can add that capability after Tim's config changes :)